packages/manager/modules/pci/src/projects/new/new.routing.js

```diff
@@ -0,0 +1,76 @@
+import find from 'lodash/find';
+import get from 'lodash/get';
+import EnvironmentService from '@ovh-ux/manager-config';
+
+import { PCI_REDIRECT_URLS } from '../../constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.new', {
+      url: '/new?description',
+      component: 'pciProjectNew',
+      resolve: {
+        paymentStatus: /* @ngInject */ $transition$ => get($transition$.params(), 'hiPayStatus')
```

Comment:
`hiPayStatus` & `paypalAgreementStatus` should be defined in state `url`, no ?

Reply:
This is done into packages/manager/modules/pci/src/projects/new/payment/payment.routing.js as it is payment information query params.
Don't know if it's more logical to set them in this file.